README.md

```diff
@@ -8,57 +8,59 @@ provided "as is"; we make no guarantees about any plugin in this repo.
 
  ![run-test](https://i.imgur.com/tKSQH5e.png)
 
- 7. Use the refactor tool to rename the package to what you want your plugin to be. Rightclick the package in the sidebar and choose *Refactor > Rename*. 
+ 6. Use the refactor tool to rename the package to what you want your plugin to be. Rightclick the package in the sidebar and choose *Refactor > Rename*. I choose to rename it to `com.helmetcheck`.
 
-  Do the same for `ExamplePlugin`, `ExampleConfig` and `ExamplePluginTest`.
+ 7. Use the same tool, *Refactor > Rename*, to rename `ExamplePlugin`, `ExampleConfig` and `ExamplePluginTest` to `HelmetCheckPlugin` etc.
  
-  Right-click the src folder in the sidebar and choose *Replace in path*, enter `example` in the upper field and `<yourpluginname>`, no spaces. in the lower one. Then press replace all. Go to your plugin file and set it's name in the `PluginDescriptor`, this can have spaces.
+ 8. Go to your plugin file and set it's name in the `PluginDescriptor`, this can have spaces.
 
  9. Open the `runelite-plugin.properties` file and add info to each row. 
  ```
-displayName=Helmet check
-author=dekvall
-support=https://github.com/dekvall/helmet-check
-description=Alerts you when you have nothing equipped in your head slot
-tags=hint,gear,head
-plugins=com.example.HelmetCheckPlugin
+ displayName=Helmet check
+ author=dekvall
+ support=https://github.com/dekvall/helmet-check
+ description=Alerts you when you have nothing equipped in your head slot
+ tags=hint,gear,head
+ plugins=com.helmetcheck.HelmetCheckPlugin
  ```
- `support` is the URL you want players to use to leave feedback for your plugin; you can just use your repository for that. `tags` will make it easier to find your plugin when searching for related words.
+ `support` is the URL you want players to use to leave feedback for your plugin; you can just use your repository for that. `tags` will make it easier to find your plugin when searching for related words. If you want to add multiple plugin files, the `plugins` field allows for comma separated values, but this is not usually needed.
 
  10. Optionally, you can add an icon to be displayed alongside with your plugin. Place a file with the name `icon.png` no larger than 48x72 px at the root of the repository.
 
- 11. When you have your plugin working. Commit your changes and push them to your repository. 
+ 11. Write a nice README so your users can see the features of your plugin.
+
+ 12. When you have your plugin working. Commit your changes and push them to your repository. 
 
 
 ### Using the script
  1. Navigate to the folder in which you cloned the `plugin-hub` repository.
 
  2. Run the script with:
  ```
- python3 create_new_plugin.py
+ python3 create_new_plugin.py [--output_directory OUTPUT_DIRECTORY]
  ```
  It will ask you a series of questions, and then generate a folder with the name of your plugin.
+ There's also a non-interactive mode, you can list the options by running `python3 create_new_plugin.py --help`.
```

Comment:
I don't think documenting the flags is particularly useful here, they are basically just so I can test it easily

Reply:
I see, yea maybe not, i'll remove it.